### DISCUSSION TITLE
[10] A "contribute" action to concore studies.

### DISCUSSION THREAD

**Discussion by parteekcoder** [ControlCore-Project/concore-editor#111] [General]
@pradeeban can you please tell me about this project

Like what work is done and what to be implemented, so that I can propose my ideas

**Comment by pradeeban** [comment]
The idea is a script in concore project (similar to the scripts we have). Then, connect that script with the "Contribute" button in the concore editor.

This is to directly contribute the concore studies you visually compose into a Git repository (typically, the git repository of concore itself, under a specific folder. Say, a folder of "community contributed."
